### ISSUE TITLE
[Bug] 'Invalid unordered_map<K, T> key' error at startup

### ISSUE THREAD

**Issue by pidgeon777** [pit-ray/win-vind#78]
**Describe the bug**

This error appears when launching the latest portable x64 version:

![image](https://user-images.githubusercontent.com/13739892/159700786-fae7f31c-0a8d-47af-a9ca-78907b6498dd.png)

**Environment**

- Windows 10 Edition: Pro
- OS Type: 64bit
- OS Version: 21H2
- OS Build: 19044.1586
- win-vind Version: v4.3.3
- win-vind Type: zip
- Is it reproducible?: Yes

**Comment by pit-ray** [comment]
Thanks for the report.

Are there any messages in the log file? If so, please share.
If you do not wish to make it public, then send it to pit-ray@outlook.com

**Comment by pidgeon777** [comment]
```log
========== System Infomation ==========
[Windows]
      Edition: Windows 10 Pro
      Version: 21H2
Build Numbers: 10.0.19044
 Architecture: x64

[win-vind]
      Version: 4.3.3
=======================================
[Message] E: Invalid Syntax (L:827)

[Error] An runtime exception occurred from class std::vector<class vind::core::KeyCode,class std::allocator<class vind::core::KeyCode> > __cdecl vind::core::parse_ascii_command(char). The character '~' is invalid ascii key code. (C:\Users\myusername\AppData\Local\Temp\b53b63a89c3b87cb5455c7485c47a8\win-vind\default_config\.vindrc, L:827) (enum vind::SystemCall __cdecl vind::bind::SyscmdSource::sprocess(const class std::filesystem::path &,bool))
```

I solved this by replacing the `~` character with `'` in the `.vindrc` file, line `827`:

```vim
" Switch char case (Vim Emulation: ')
ennoremap ' switch_char_case
```

**Comment by pit-ray** [comment]
Can you tell me the language of the system and the keyboard layout?
I would like to reproduce it.

Thanks again.

**Comment by pidgeon777** [comment]
System Language: Italian
Keyboard Layout: Italian - QWERTY

**Comment by pit-ray** [comment]
Thanks. I try that setting.

**Comment by pit-ray** [comment]
Sorry for the late reply.

I was able to reproduce this when I started win-vind using an Italian layout keyboard.

win-vind can only map characters that can be typed with a simple key sequence. This seems to have caused a mapping error.

In Italian layout, how does `~` in Vim generally map?

For now, I am considering not changing the default value, outputting the error to the log, and ignoring it.

Thanks.

**Comment by Rain-ctrl** [comment]
Has anyone managed to solve this problem with the Italian layout keyboard?

**Comment by pit-ray** [comment]
@Rain-ctrl 
I do not have an Italian layout keyboard and am unable to confirm the key code. 
How do you type `~` when using Vim with an Italian keyboard?

**Comment by Rain-ctrl** [comment]
Hi @pit-ray how are you? With the Italian keyboard we have to press the "alt" + "126" key

**Comment by pit-ray** [comment]
@Rain-ctrl 
Thanks for the info.

I checked and it seems that the current Windows standard Italian layout does not allow `~` to be entered oneshot. win-vind prefers one-shot input when describing with letters, so I assume it is an error.
https://learn.microsoft.com/en-us/globalization/keyboards/kbdit.html,

Instead, Italian (142) allows one-shot input of `~` and win-vind starts normally.
https://learn.microsoft.com/en-us/globalization/keyboards/kbdit142.html

I will consider solution.

Thanks.

**Comment by pit-ray** [comment]
Fixed at v5.2.3.
